### PR TITLE
Add Covid19 surveillance data

### PR DESCRIPTION
**Pull request by kkothari2001**
Adds the dataset requested in https://github.com/weecology/retriever/issues/1582

New dataset is named covid-case-surveillance

Tested with `retriever ls` and install to postgres. 

Note: Many of the data values seen might be NA but that is just the nature of the first few rows of data.

### CODE REVIEW

File: scripts/covid_case_surveillance.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+    "citation": "not-sure-what-to-fill",
```
Comment (henrykironde) [inline scripts/covid_case_surveillance.json]: leave the citation as ""

**Review by henrykironde — COMMENTED** [review]
Hi @kkothari2001, the reason why we made this wait was to verify if the link is stable.
It seems the link is table and  ready to be merged. Could you please change the citation value  and this will be ready. 
Thanks you much for the contribution

**Comment by henrykironde** [comment]
Something strange is happening to the version.txt file

**Comment by henrykironde** [comment]
Try to first update this branch with main and see if it still happens

**Comment by kkothari2001** [comment]
Solved it @henrykironde there was an issue in detecting python scripts.

**Pull request merged**